DS/Assignment2/mycollectionclasses/CollectionsMain.java

```diff
@@ -0,0 +1,60 @@
+package mycollectionclasses;
+
+import java.util.Scanner;
+
+import mycollectionclasses.college.counseling.Councellor;
+
+public class CollectionsMain {
+
+    public static void main(String[] args) {
+	Stacks();
+	infixToPostfix();
+	Queues();
+	councellor();
+    }
+
+    public static void Queues() {
+	Queue queue = new Queue();
+	queue.enqueue(1);
+	queue.enqueue(2);
+	queue.enqueue(3);
+	queue.display();
+	queue.dequeue();
+	queue.dequeue();
+	System.out.println(queue.getFront());
+	queue.makeEmpty();
+	queue.display();
+    }
+
+    public static void Stacks() {
+	Stack stack = new Stack();
+	stack.push(1);
```

Comment:
Is there any specific to add values 10, 11 etc to stack? If not use the loop for insertion